src/Sql/dbo/User Defined Types/OrganizationUserType.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TYPE [dbo].[OrganizationUserType] AS TABLE(
```

Comment:
So I see the UDT, but I would think Dapper could handle mapping the table type column names with the C# class construct automatically w/o explicit ordinals.

Reply:
I ended up with SQL conversion errors attempting to cram strings into DateTimes and vice-versa.